### PR TITLE
Rename `perBlock` to `perSecond`

### PR DESCRIPTION
**Pull request by shahthepro**
## Code Change Checklist

To be completed before internal review begins:

- [ ]  The contract code is complete
- [ ]  Executable deployment file
- [ ]  Fork tests that test after the deployment file runs
- [ ]  Unit tests *if needed
- [ ]  The owner has done a [full checklist review](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review.md) of the code + tests

Internal review:

- [ ] Two approvals by internal reviewers

### CODE REVIEW

File: contracts/contracts/harvest/Dripper.sol [hunk 1]
```diff
@@ -49,7 +49,7 @@ contract Dripper is Governable {
 
     struct Drip {
         uint64 lastCollect; // overflows 262 billion years after the sun dies
-        uint192 perBlock; // drip rate per block
+        uint192 perSecond; // drip rate per block
```
Comment (naddison36) [inline contracts/contracts/harvest/Dripper.sol:52]: nit the comment can also be updated to `per second`

**Review by naddison36 — DISMISSED** [review]
I like how you also regenerated the contract diagrams. Thanks

File: contracts/contracts/harvest/Dripper.sol (hunk 1 above)
Comment (shahthepro) [reply contracts/contracts/harvest/Dripper.sol]: Thanks, have fixed the comment

**Comment by shahthepro** [comment]
Moving this changeset to PR #2248

**Pull request closed without merging**